examples/dlb/example.rst

```diff
@@ -0,0 +1,105 @@
+.. _install_sandboxes_dlb:
+
+DLB
+===
+
+By enabling connection balancer in Envoy you can balance the connections cross the threads and improve performance.
+
+This sandbox provides an example about how to enable DLB connection balanace.
+
+.. note::
+   Please run below command to check your CPU supports DLB:
+
+   .. code-block:: console
+
+   $ lspci -d :2710
+
+   You should get output like below:
+
+   .. code-block:: text
+
+      5a:00.0 Co-processor: Intel Corporation Device 2710
+      6b:00.0 Co-processor: Intel Corporation Device 2710
+      7c:00.0 Co-processor: Intel Corporation Device 2710
+      8d:00.0 Co-processor: Intel Corporation Device 2710
+
+   The number and PCIE address of DLB devices vary from CPU to CPU.
+
+   Also check your kernel version, 5.15+ is good.
```

Comment:
this would also require that any relevant kernel modules have been compiled in

surely there is a more reliable way of checking whether a platform has DLB support - esp given the above deviced id is not reliable

Reply:
The kernel module is installed by hand, if devices exist, means the platform supports DLB.